README.md

```diff
@@ -70,11 +70,12 @@ Configures [moo](https://github.com/no-context/moo) tokenizer for specific langu
 - Identifiers: `foo`, `Bar`, `_baz123`, etc
 - Line joins: if the line ends with `\`, the next one will be treated as its continuation
 
-You can refer to `LexerConfig` interface for more details. Also check out [usage example for Python](https://github.com/renovatebot/parser-utils/blob/main/lib/lang/python.ts).
+Refer to the `LexerConfig` interface for more details.
+Also check out [our usage example for Python](https://github.com/renovatebot/parser-utils/blob/main/lib/lang/python.ts).
 
 ### [`lib/parser`](https://github.com/renovatebot/parser-utils/tree/main/lib/tree)
 
-This layer responsible for transforming token sequence to the nested tree with the tokens as leafs.
+This layer is responsible for transforming the token sequence to the nested tree with the tokens as leafs. tree with the tokens as leafs.
```

Comment:
```suggestion
This layer is responsible for transforming the token sequence to the nested tree with the tokens as leafs.
```

I think something went wrong when editing the line?

Please check if the suggestion is correct before applying it.

Reply:
Thanks, it was my VSCode in-line PR view